frontend/apps/marketing/src/components/contentful/activityCatalog/facetBar/facetBar.tsx

```diff
@@ -69,20 +73,51 @@ const FacetBar = ({
       return (
         <FormControl
           key={facet}
-          sx={{minWidth: 0, display: isInDrawer ? 'block' : 'inline-flex'}}
+          sx={{
+            minWidth: 0,
+            '&:not(:last-of-type)': {mb: 1.25, mr: 2},
+          }}
         >
           <Select
             aria-label={FACET_LABELS[facet]}
             multiple
             onChange={e => handleChange(facet, e.target.value[0])}
             value={[]}
+            // NEW: control open/close + close on menu mouse leave
+            open={openFacet === facet}
+            onOpen={() => setOpenFacet(facet)}
+            onClose={() => setOpenFacet(null)}
+            MenuProps={{
+              disablePortal: true,
+              disableScrollLock: true,
+              anchorOrigin: {vertical: 'bottom', horizontal: 'left'},
+              transformOrigin: {vertical: 'top', horizontal: 'left'},
+              PaperProps: {
+                onMouseLeave: () => setOpenFacet(null),
```

Comment:
You'll need to confirm that this works on keyboard and mobile phone since there is no mouse leave on phones.